Formula/php.rb

```diff
@@ -416,3 +433,33 @@ def plist; <<~EOS
     end
   end
 end
+
+__END__
+diff --git a/acinclude.m4 b/acinclude.m4
+index 1deb50d2983c..d0e66c8b6344 100644
+--- a/acinclude.m4
++++ b/acinclude.m4
+@@ -441,7 +441,11 @@ dnl
+ dnl Adds a path to linkpath/runpath (LDFLAGS)
+ dnl
+ AC_DEFUN([PHP_ADD_LIBPATH],[
+-  if test "$1" != "/usr/$PHP_LIBDIR" && test "$1" != "/usr/lib"; then
++  case "$1" in
++  "/usr/$PHP_LIBDIR"|"/usr/lib"[)] ;;
++  /Library/Developer/CommandLineTools/SDKs/*/usr/lib[)] ;;
++  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/*/usr/lib[)] ;;
```

Comment:
You'll want to try adding an `*` after Xcode here to exclude the beta app's path too.

Reply:
Yeah I haven't gotten there yet since I'm still fiddling with getting the configure section to pass =) Good point though.